lib/quick_travel/booking.rb

```diff
@@ -60,9 +60,10 @@ def self.create(options = {})
 
     # Update an existing booking
     def update(attrs = {}, options = {})
-      response = put_and_validate("#{api_base}/#{@id}.json", options.merge(booking: attrs))
+      response_object = put_and_validate("#{api_base}/#{@id}.json", options.merge(booking: attrs), return_response_object: true)
+      response = response_object.parsed_response
       # id is returned if other attributes change otherwise success: true
-      fail AdapterError.new(response) unless response['id'] || response['success']
+      fail AdapterError.new(response) unless response_object.no_content? || response['id'] || response['success']
```

Comment:
Is this because we are returning a 204 OK from QT now?

Reply:
Yes